ADOPTERS.MD

```diff
@@ -31,7 +31,7 @@ If you would like to be included in this table, please open [an issue](https://g
 | ManoMano*                            |                                                                      | Development & Production | [link](https://www.manomano.es/)                                         |
 | Optum*                               |                                                                      | Production               | [link](https://optum.com/)                                               |
 | Red Hat                              | Red Hat OpenShift Serverless                                         | Generally Available      | [link](https://www.openshift.com/learn/topics/serverless)                |
-| Run:AI                               | Run:AI Platform                                                      | Production               | [link](https://www.run.ai/platform)                                      |
+| Run:ai                               | Run:AI Platform                                                      | Production               | [link](https://www.run.ai)                                               |
```

Comment:
Sorry, but "Run:AI Platform" should also be changed to "Run:ai Platform". Thank you

Reply:
oh whoops - I didn't push my latest changes. Let me do a follow up

Reply:
done: https://github.com/knative/community/pull/1653